examples/plotting/file/line_on_off.py

```diff
@@ -37,9 +26,10 @@
 l1 = p.line(x, 4 * np.cos(x), color=Viridis3[1], legend="Line 1", **props)
 l2 = p.line(x, np.tan(x), color=Viridis3[2], legend="Line 2", **props)
 
-callback = CustomJS(code=code, args={})
-checkbox = CheckboxGroup(labels=["Line 0", "Line 1", "Line 2"], active=[0, 1, 2], callback=callback, width=100)
-callback.args = dict(l0=l0, l1=l1, l2=l2, checkbox=checkbox)
+checkbox = CheckboxGroup(labels=["Line 0", "Line 1", "Line 2"],
+                         active=[0, 1, 2], width=100)
+checkbox.callback = CustomJS(args=dict(l0=l0, l1=l1, l2=l2, checkbox=checkbox),
+                             **code)
```

Comment:
I'm -0.03 on `**code`.  That is to say, it's not to my taste, and I think I have a good reason for it. But not to my taste is not a reason to block code.

In case you're interested: I use `**props` when I need to use the same arguments over and over again to save code and maintenance. This isn't true in this case and I think it makes it harder to read what are arguments for CustomJS and what aren't, because the `code` definition is separated from the call to CustomJS. In addition, I remember finding ** confusing as a python beginner and having the splat be named the same thing as an argument I think is confusing.